### ISSUE TITLE
Invalid Header Failure with Chef Environment

### ISSUE THREAD

**Issue by tacchino** [hashicorp/terraform#8382]
Receiving a failure with chef_environment on Terraform v0.7.1 during a terraform plan.

```
Error refreshing state: 1 error(s) occurred:

* chef_environment.env: Get https://api.opscode.com/organizations/xxx/environments/staging: net/http: invalid header field name "Hashed Path"
```

Works fine with v0.7.0

* Terraform v0.7.1
* chef_environment

**Comment by donaldguy** [comment]
I saw the same thing in `TF_LOG=INFO output` of a (still running) `terraform refresh` but with respect to a data bag.

**Comment by donaldguy** [comment]
Seems like the vendor needs to be updated to pull in go-chef/chef#75

seems to relate to switch from a go1.5 to a go1.6 compilation?

**Comment by mitchellh** [comment]
Fixed by #8401
